main.rb

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/ruby
```

Comment:
I hope you know writing this file and setting up a command-line interface was above and beyond what we asked for. 
But glad you were able to do it. :)